docker-autoscale/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker-private.infra.cloudera.com/cloudera_base/cldr-java:11.0.13-cldr-jre-slim-buster-15122021
+FROM docker-private.infra.cloudera.com/cloudera_base/ubi8/cldr-ubi-minimal-java:8.4-208
```

Comment:
Note: The latest is `cldr-ubi-minimal-java:jdk11-ubi8.5-218` (January 19, 2022 21:55).

Reply:
The pull request had been opened earlier 😆

Reply:
Good point. :smile: